src/javascripts/dropdown.js

```diff
@@ -0,0 +1,26 @@
+// Opens respective dropdown when clicked
+(function () {
+
+	// Gather elements
+	dropdown_toggle = document.getElementsByClassName('dropdown__toggle');
+
+	// Add listener to dropdowns
+	for (var i = 0; i < dropdown_toggle.length; i++) {
+		dropdown_toggle[i].addEventListener('click', toggleDropdown, false);
+	}
+
+	// Add / Remove hidden class
+	function toggleDropdown(){
+		// Get parent of dropdown
+		dropdown_parent = this.parentElement;
+		dropdown_content = dropdown_parent.querySelector('.dropdown__content');
+
+		//Check dropdown content
+		if (dropdown_content.className.includes('hidden')) {
```

Comment:
The `includes` method is not supported below IE12. Alternative would be to use `classList.contains('hidden')`